api/ImageData.json

```
@@ -148,6 +148,68 @@
                 "deprecated": false
               }
             }
+          },
+          "pixelFormat_option": {
+            "__compat": {
+              "spec_url": "https://html.spec.whatwg.org/multipage/canvas.html#canvascolortype",
+              "support": {
+                "chrome": {
+                  "version_added": "137"
+                },
+                "chrome_android": "mirror",
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": "mirror",
+                "safari": {
+                  "version_added": false
```

Safari has a tracking bug:

```suggestion
                  "version_added": false,
                  "impl_url": "https://webkit.org/b/291196"
```